push-down-test/run-test.sh

```diff
@@ -0,0 +1,366 @@
+# GO environment variables
```

Comment:
revert the rename so that github diff will work

Reply:
Agreed